scripts/tests/service_tests.py

```diff
@@ -40,7 +40,7 @@ def test_agent_go_net_http(self):
                         ELASTIC_APM_FLUSH_INTERVAL: 500ms
                         ELASTIC_APM_SERVICE_NAME: gonethttpapp
                         ELASTIC_APM_TRANSACTION_IGNORE_NAMES: healthcheck
-                        ELASTIC_APM_VERIFY_SERVER_CERT: True
+                        ELASTIC_APM_VERIFY_SERVER_CERT: 'True'
```

Comment:
I think the rule is that agents only allow lowercase `true` and `false` as boolean string values. But I could be mistaken... at least that's what the Node.js agent currently requires